### PR TITLE
im not sure whats happened, i think i have 2 identical repositories a…

### PR DESCRIPTION
**Pull request by jondon94**
…nd ive been uploading to the wrong one, here is all the latest stuff, all working, further improvements to come

### CODE REVIEW

File: profileAbout.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>About</title>
+  </head>
+
+  <main class="backDrop">
+
+  <body>
```
Comment (JSChilds) [inline profileAbout.html]: The body tag should not be used more than once and shouldn't be nested inside any tag other than html. All the content on a page should be inside the body. For example:

<html>
    <body>
        All the page elements and content goes in here.
    </body> 
</html>

File: prostyle.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+.bio {
+  background-color: darkgray;
+  width: 500px; height: auto;
+  margin-left: 480px;
+  border-style: outset;
+  font-family: serif;
+  font-size: 18px;
+}
+
+.intrests {
```
Comment (JSChilds) [inline prostyle.css]: Should be spelt "interests" - be careful with spelling, as you may find yourself unable to target things you think you should be able to.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Change your html to only have one body tag that all the rest of the page content is nested inside.

File: profileAbout.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>About</title>
+  </head>
+
+
+<!-- <div class="container-fluid">
```
Comment (JSChilds) [inline profileAbout.html]: Try to avoid leaving in commented-out code.

File: profileSkills.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="prostyle.css">
+
+    <title>Skills</title>
+
+  <main class="backDrop">
+  </head>
+
```
Comment (JSChilds) [inline profileSkills.html]: Try to keep spacing consistent when you use it.

File: profileAbout.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  <div class="row">
+    <div class="col-3">
+    </div>
+  </div>
+</div> -->
+
+
+  <div class="backDrop">
+
+  <body>
```
Comment (JSChilds) [inline profileAbout.html]: There should only be one body tag that all the page content is nested inside.

**Review by JSChilds — CHANGES_REQUESTED** [review]
The body tag issues still need to be changed, but other than that very nice!

**Review by richgurney — CHANGES_REQUESTED** [review]
Can you change profileHome.html to index.html. To put this on GitHub you will need to have the correct name for the deployment to work

File: profileAbout.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>About</title>
+  </head>
+
+  <body class="backDrop">
+
```
Comment (JSChilds) [inline profileAbout.html:11]: Everything from here to the end of body should be indented.

File: profileContact.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>Contact</title>
+  <main class="backDrop">
+  </head>
+
+  <body>
```
Comment (JSChilds) [inline profileContact.html:11]: Body tag shouldn't be here.

File: profileHome.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="prostyle.css">
+    <title>Portfolio: Jonathan Donaldson</title>
+  </head>
+
+  <main class="backDrop">
+
+  <body>
```
Comment (JSChilds) [inline profileHome.html:13]: Body tag shouldn't be here.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Still seeing a couple uses of body tags that shouldn't be there. Remember - there should be only one body tag and all the page content should be nested inside it.

**Pull request merged**